website/docs/getting-started/setup/installation-guides/azure-aci.md

```diff
@@ -0,0 +1,130 @@
+---
+
+description: Deploy Appsmith on Azure Container Instance Service
+sidebar_position: 4
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Azure Container Instance
+Azure Container Instances (ACI) is a simple and efficient way to run containers in the cloud. This document guides you through launching an ACI instance and running an Appsmith container. The data persists in Azure Storage Account File Share.
+
+:::note 
+Azure only supports CIFS file shares and doesn't support NFS file shares.
+:::
+
+### Prerequisites​
+
+Before launching an ACI instance, you need to have an Azure subscription and have the Azure CLI installed on your machine. 
+
+* [Azure Subscription](https://azure.com/free) - If you don't have an Azure subscription, you can sign up for a free trial
+* Azure CLI 
```

Comment:
@sum35h can we link Azure CLI to https://learn.microsoft.com/en-us/cli/azure/  ?

Reply:
Sure, lets add it.

Reply:
Sure, I'm adding it.